cabal.project

```diff
@@ -12,10 +12,13 @@ repository cardano-haskell-packages
 
 -- See CONTRIBUTING for some Nix commands you will need to run if you
 -- update either of these.
--- Bump this if you need newer packages from Hackage
-index-state: 2022-12-01T00:00:00Z
--- Bump this if you need newer packages from CHaP
-index-state: cardano-haskell-packages 2022-11-15T00:00:00Z
+-- repeat the index-state for hackage to work around haskell.nix parsing limitation
+index-state: hackage.haskell.org 2022-12-01T00:00:00Z
+index-state: 
+  -- Bump this if you need newer packages from Hackage
+  , hackage.haskell.org 2022-12-01T00:00:00Z
```

Comment:
Ah, Nix CI is failing due to our hackage.nix pin being too old, which was only possible to pass CI due to the bug you are fixing here. So bumping hackage.nix should get CI on green :+1:

Reply:
There are now CI failures due to the new HLS version, maybe revert hackage.nix index state back to the first of December to unblock this PR?